### PR TITLE
Add layout for the MiniTrend on downscaling page

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
Layout for the downscaling page including the three columns that hold a mini bar chart.
The bar chart is already reviewed in another pull request, but here it is combined ready to go to the downscaling page

### CODE REVIEW

File: packages/app/src/pages/afschaling.tsx [hunk 1]
```diff
@@ -10,9 +17,20 @@ import {
   StaticProps,
 } from '~/static-props/create-get-static-props';
 import { getLastGeneratedDate } from '~/static-props/get-data';
+import { replaceVariablesInText } from '~/utils/replaceVariablesInText';
 
 export const getStaticProps = createGetStaticProps(getLastGeneratedDate);
 
+// CONNECT WITH REAL DATA
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/pages/afschaling.tsx:24]: add `@TODO` to find this easier later on

File: packages/app/src/pages/afschaling.tsx [hunk 2]
```diff
@@ -47,10 +65,112 @@ const Afschaling = (props: StaticProps<typeof getStaticProps>) => {
               />
             </Box>
           )}
+
+          <Box
+            display="grid"
+            gridTemplateColumns={{ _: undefined, md: 'repeat(3, 1fr)' }}
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/pages/afschaling.tsx:71]:
```suggestion
            gridTemplateColumns={{ md: 'repeat(3, 1fr)' }}
```

File: packages/app/src/pages/afschaling.tsx [hunk 3]
```diff
@@ -47,10 +65,112 @@ const Afschaling = (props: StaticProps<typeof getStaticProps>) => {
...
+            css={css({ columnGap: 5 })}
+            mb={5}
+          >
+            <MiniTrend
+              title="Reproductiegetal"
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/pages/afschaling.tsx:76]: Add these titles to the locale file?

File: packages/app/src/pages/afschaling.tsx [hunk 4]
```diff
@@ -47,10 +65,112 @@ const Afschaling = (props: StaticProps<typeof getStaticProps>) => {
...
+              icon={Repro()}
+              is_below_threshold={reproduction_is_below_threshold}
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/pages/afschaling.tsx:78]: props should be camel case

File: packages/app/src/pages/afschaling.tsx [hunk 5]
```diff
@@ -47,10 +65,112 @@ const Afschaling = (props: StaticProps<typeof getStaticProps>) => {
...
+}
+
+function MiniTrend({
+  title,
+  icon,
+  is_below_threshold,
+  threshold_day_span,
+  children,
+}: MiniTrendProps) {
+  const descriptionTrue =
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/pages/afschaling.tsx:123]: more locale

File: packages/app/src/pages/afschaling.tsx [hunk 6]
```diff
@@ -47,10 +65,112 @@ const Afschaling = (props: StaticProps<typeof getStaticProps>) => {
...
+    'Waarde is minder dan {{days}} dagen onder de grenswaarde';
+  const descriptionFalse = 'Grenswaarde is overschreden';
+
+  return (
+    <Box mb={{ _: 3, md: 0 }}>
+      <Box display="flex" alignItems="center" mb={1}>
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/pages/afschaling.tsx:129]: is there really a need for so many display: flex in this structure?

File: packages/app/src/pages/afschaling.tsx [hunk 7]
```diff
@@ -47,10 +65,112 @@ const Afschaling = (props: StaticProps<typeof getStaticProps>) => {
...
+        </Text>
+      </Box>
+      <Box display="flex" mb={3}>
+        <Box
+          height={18}
+          width={18}
+          display="flex"
+          alignItems="center"
+          justifyContent="center"
+          backgroundColor={is_below_threshold ? '#0390D6' : 'red'}
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/pages/afschaling.tsx:144]: colors should reference the theme

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Comment by VWSCoronaDashboard1** [comment]
One general QA comment I suppose:

image

I guess it would look nicer if charts are aligned at the bottom

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Review by VWSCoronaDashboard10 — APPROVED** [review]

**Review by VWSCoronaDashboard8 — APPROVED** [review]

**Pull request merged**